core/src/main/java/cz/xtf/core/bm/BuildManager.java

```diff
@@ -28,7 +28,7 @@ public BuildManager(OpenShift openShift) {
                 // Otherwise you can see:
                 // $ oc label namespace <name> "label1=foo"
                 // Error from server (Forbidden): namespaces "<name>" is forbidden: User "<user>" cannot patch resource "namespaces" in API group "" in the namespace "<name>"
-                OpenShifts.admin().namespaces().withName(openShift.getNamespace())
+                OpenShifts.admin(openShift.getNamespace()).namespaces().withName(openShift.getNamespace())
```

Comment:
Must caller of `OpenShifts.admin()` provide namespace. Couldn't `OpenShifts.admin()` find out itself correct namespace?